packages/app-page-builder-elements/src/renderers/heading.tsx

```diff
@@ -16,7 +17,18 @@ export const createHeading = () => {
         const __html = element.data.text.data.text;
 
         if (isValidLexicalData(__html)) {
-            return <LexicalHtmlRenderer theme={theme} value={__html} />;
+            return (
+                <LexicalHtmlRenderer
+                    theme={theme}
+                    themeStylesTransformer={styles => {
```

Comment:
When I try to build the project shows `Parameter 'styles' implicitly has an 'any' type.`.